vivarium/core/store.py

```diff
@@ -1064,13 +1073,14 @@ def _delete_path(self, path):
         if not path:
             self.inner = {}
             self.value = None
-            return self
+            return None
         target = self.get_path(path[:-1])
         remove = path[-1]
         if remove in target.inner:
-            lost = target.inner[remove]
+            # End any parallel processes to be deleted
+            self.recursive_end_process(target.inner[remove])
             del target.inner[remove]
-            return lost
+            return None
```

Comment:
likewise here, is `lost` deleted? If it is not, then we should try to keep the prior API

Reply:
Same as above.